.github/workflows/ci_win.yml

```diff
@@ -95,23 +95,23 @@ jobs:
         ln -s ${{ env.FFMPEG_SRC }} ffmpeg
 
     - name: Setup
-      shell: cmd
+      shell: pwsh
       run: |
-        "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Auxiliary\Build\vcvars64.bat" x64 && ^
-        meson setup builddir --backend vs --buildtype release --default-library ${{ matrix.build_type }} ^
+        .\scripts\msvc-env.ps1
+        meson setup builddir --backend vs --buildtype release --default-library ${{ matrix.build_type }} `
         --pkg-config-path "D:\a\nope.media\nope.media\ffmpeg\lib\pkgconfig"
 
     - name: Build
-      shell: cmd
+      shell: pwsh
       run: |
-        "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Auxiliary\Build\vcvars64.bat" x64 && ^
+        .\scripts\msvc-env.ps1
         meson compile -C builddir
 
     - name: Test
-      shell: cmd
+      shell: pwsh
       run: |
-        set PATH=D:\a\nope.media\nope.media\ffmpeg\bin;%PATH%
-        "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Auxiliary\Build\vcvars64.bat" x64 && ^
+        .\scripts\msvc-env.ps1
+        $env:Path += ";D:\a\nope.media\nope.media\ffmpeg\bin"
```

Comment:
shouldn't you prepend this one as well so it takes priority over what's installed by the CI (like originally)? Typically in case the Github Actions start deploying their own version of FFmpeg.

Reply:
Done in a separate commit.

Reply:
Why in a separate commit? The original code was prepending 🤔

Reply:
Fixed up.